cdap-security/src/main/java/co/cask/cdap/security/guice/SecureStoreModules.java

```diff
@@ -59,14 +58,22 @@
  *
  */
 public class SecureStoreModules extends RuntimeModule {
+  public static final String DELEGATE_SECURE_STORE = "delegateSecureStore";
+  public static final String DELEGATE_SECURE_STORE_MANAGER = "delegateSecureStoreManager";
 
   @Override
   public final Module getInMemoryModules() {
     return new AbstractModule() {
       @Override
       protected void configure() {
-        bind(SecureStore.class).toProvider(new TypeLiteral<StoreProvider<SecureStore>>() { });
-        bind(SecureStoreManager.class).toProvider(new TypeLiteral<StoreProvider<SecureStoreManager>>() { });
+        bind(SecureStore.class)
```

Comment:
Should use `PrivateModule` instead and have these two `DELEGATE` binding as private binding and only exposes the unannotated `SecureStore` and `SecureStoreManager`.